Darkly/Models/DataManager.m

```diff
@@ -66,11 +66,14 @@ -(void) saveUser: (LDUser *) user {
         [[DataManager sharedManager] purgeOldUsers];
```

Comment:
In DataManager, sometimes you use [DataManager sharedManager] instead of self. Also, in at least one method, DataManager.purgeOldUsers, you are saving the context outside of the block.